apps/sms/views/inbox/js/inbox.js

```diff
@@ -146,7 +149,7 @@ var InboxView = {
     var draftId = node.dataset.draftId;
 
     var threadOrDraft = draftId ?
-      Drafts.byDraftId(+draftId) : Threads.get(threadId);
+      Drafts.byDraftId(+draftId) : Threads.get(+threadId);
```

Comment:
Yeah, but the problem is that in some places in inbox.js Threads.get/has is used with numbers and in some with strings. I made this change just to simplify tests, otherwise I'll have to write smth like this:
```
Threads.get.withArgs(thread.id).returns(thread);
Threads.get.withArgs(thread.id.toString()).returns(thread);
// or Threads.get.withArgs(sinon.match(thread.id)).returns(thread); that uses == internally
Threads.has.withArgs(thread.id).returns(true);
Threads.has.withArgs(thread.id.toString()).returns(true);

// instead of just

Threads.get.withArgs(thread.id).returns(thread);
Threads.has.withArgs(thread.id).returns(true);
```

Ideally I'd rather always pass the same type everywhere, but tell me if you prefer tests with additional ```.toString()``` or ```sinon.match(thread.id)``` stubs?